src/content/doc-surrealql/clauses/index.mdx

```diff
@@ -0,0 +1,23 @@
+---
+sidebar_position: 1
+sidebar_label: Clauses
+title: Clauses | SurrealQL
+description: Clauses are used to 
+---
+
+# Clauses
+
+In SurrealQL, clauses can be used to alter the way a query is executed. They are used in the following ways: 
+
+- [`EXPLAIN`](/docs/surrealql/clauses/explain): Explain the query plan.
+- [`FETCH`](/docs/surrealql/clauses/fetch): Fetch related records.
```

Comment:
How about something like "Fetch all the fields of related records"? i.e. if you have a bunch of record IDs then FETCH makes the output more meaningful, especially if the records are random IDs.